src/LocaleProvider.tsx

```diff
@@ -7,7 +8,19 @@ import {
   DEFAULT_REGION,
   FALLBACK_LANGUAGE,
 } from '@atb/translations/commons';
-import { AppState } from 'react-native';
+import {AppState} from 'react-native';
+
+let globalLanguage: Language = DEFAULT_LANGUAGE;
+export const getGlobalLanguage = () => globalLanguage;
+const setGlobalLanguage = (language: Language) => {
+  globalLanguage = language;
+};
+/**
+ * tGlobal can be used instead of the t function for when you don't want
+ * language changes to potentially retrigger an action (such as e.g. an alert box)
+ */
+export const tGlobal: TFunc<typeof Language> = (translatable) =>
+  translatable[globalLanguage];
```

Comment:
@gorandalum Do you think this should use `getGlobalLanguage()` instead, allowing `tGlobal` to be defined wherever is most natural?

Reply:
I think maybe it's "a bit much" to insist on functions also having to go through getters?
I quite like the last example, thinking that the "Global" naming convention is sufficiently clear.

Reply:
Lets try without and we'll get a feel for it